scispacy/linking.py

```diff
@@ -62,20 +64,24 @@ class EntityLinker:
 
     def __init__(
         self,
+        nlp: Language = None,  # required arg for spacy
+        name: str = "",  # required arg for spacy
```

Comment:
same here, can these be their actual name, or are they overridden or something?

Reply:
same as above, i can change them to be the actual names if you think that is nicer

Reply:
Ya names plz 🙏